README.md

```diff
@@ -9,9 +9,16 @@ Available under the terms of the GNU Affero General Public License. See [LICENSE
 This library targets the latest stable Rust. At time of writing, that is 1.40.0
 
 ```sh
+$ sudo apt install libpq-dev
```

Comment:
The pre-reqs section should be separate from build commands. (i.e. you don't need to install libpq-dev every time to build)

Would also include language like "On a Debian-based OS, it would be ..."